KenticoCloud.Delivery/InlineContentItems/InlineContentItemsProcessor.cs

```diff
@@ -121,5 +90,36 @@ private static List<IElement> GetInlineContentItemElements(AngleSharp.Dom.Html.I
                 .GetElementsByTagName("object")
                 .Where(o => o.GetAttribute("type") == "application/kenticocloud" && o.GetAttribute("data-type") == "item")
                 .ToList();
+
+        private string GetFragmentText(object inlineContentItem)
+        {
+            var inlineContentItemType = inlineContentItem?.GetType() ?? typeof(UnretrievedContentItem);
+            if (_inlineContentItemsResolvers.TryGetValue(inlineContentItemType, out var inlineContentItemResolver))
+            {
+                return inlineContentItemResolver(inlineContentItem);
+            }
+
+            var data = new ResolvedContentItemData<object> { Item = inlineContentItem };
+            if (_inlineContentItemsResolvers.TryGetValue(typeof(object), out var defaultContentItemResolver))
+            {
+                return defaultContentItemResolver(data);
+            }
+
+            throw new NotSupportedException("Default inline content item resolver for non specific content type was not registered.");
```

Comment:
I guess this addresses #145 .
What would you say if we did the same as in the `ReplaceElementWithFragmentNodes` method? I mean we could return the error within the text.
This would let the developer know something's wrong but the application wouldn't break completely when a new content type is added/used...
@matus12 @PetrSvirak @ondrabus ?